src/ifcconvert/SvgSerializer.cpp

```diff
@@ -66,14 +68,77 @@ void SvgSerializer::write(path_object& p, const TopoDS_Wire& wire) {
 
 	bool first = true;
 	util::string_buffer path;
-	path.add("            <path style=\"stroke:black; fill:none;\" d=\"");
 	for (TopExp_Explorer edges(wire, TopAbs_EDGE); edges.More(); edges.Next()) {
 		const TopoDS_Edge& edge = TopoDS::Edge(edges.Current());
 
 		double u1, u2;
 		Handle(Geom_Curve) curve = BRep_Tool::Curve(edge, u1, u2);
 
-		const bool reversed = edge.Orientation() == TopAbs_REVERSED;
+        Handle(Standard_Type) ty = curve->DynamicType();
+        bool conical = (ty == STANDARD_TYPE(Geom_Circle) || ty == STANDARD_TYPE(Geom_Ellipse));
+        bool closed = ALMOST_THE_SAME(u1 + PI2, u2);
+        if (conical && closed) {
+            if (first) {
+                if (ty == STANDARD_TYPE(Geom_Circle)) {
+                    Handle(Geom_Circle) circle = Handle(Geom_Circle)::DownCast(curve);
+                    double r = circle->Radius();
+                    gp_Circ c = circle->Circ();
+                    gp_Pnt center = c.Location();
```

Comment:
Small note: `gp_Cirl::Location()` returns type `const gp_Pnt &`. `gp_Pnt center = c.Location();` will make a copy, while `const gp_Pnt& center = c.Location();` will more efficiently point to the same reference. In practice though, the compiler might be smart enough not to make a copy anyway.